src/features/defi/components/Overview/Overview.tsx

```diff
@@ -34,6 +36,9 @@ export type AssetWithBalance = {
 type OverviewProps = {
   accountId?: Nullable<AccountId>
   onAccountIdChange?: (accountId: AccountId) => void
+  // The LP asset this opportunity represents
+  underlyingAsset?: AssetWithBalance
+  // The assets underlying the LP one
```

Comment:
That you felt the need to comment on these might indicate that we need to improve their naming.

Maybe replace `underlyingAsset` with `underlyingLpAsset`, or just `lpAsset`?

Reply:
32e0357ecb92199ceb456d245b8578163a11b590